src/constants/settings.ts

```diff
@@ -0,0 +1,84 @@
+import {ComponentType, SVGProps} from "react";
+import {GeneralSettings, Participants, SettingsAppearance, SettingsFeedback, Share, FileDefault} from "components/Icon";
+import {Color} from "./colors";
+
+export const MOBILE_BREAKPOINT = 920;
+
+export type MenuKey = "board" | "participants" | "appearance" | "share" | "export" | "feedback" | "profile";
+
+type LocalizationKey = "BoardSettings" | "Participants" | "Appearance" | "ShareSession" | "ExportBoard" | "Feedback" | "Profile";
+
+export type MenuItem = {
+  localizationKey: LocalizationKey;
+  location: string;
+  isModeratorOnly: boolean;
+  color: Color;
+  icon: ComponentType<SVGProps<SVGSVGElement>> | "profile"; // profile: special case
+};
```

Comment:
I think I'd prefer the following naming: `MenuItemKey`, `MenuItemConfig`

Reply:
renamed 👍